src/runtime/vm/executable.cc

```diff
@@ -525,12 +521,10 @@ void Executable::LoadConstantSection(dmlc::Stream* strm) {
   }
```

Comment:
do the size_t references above here also need to be patched?

Reply:
Here we use `uint64_t` to read `number of constants` from the Stream.
After that we read all constants from the Stream one after another.
`constants` vector size can not be bigger that `size_t` on a particular platform.
It is ok to use `size_t` here (just for looping purpose).